packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -442,7 +442,7 @@ const Conversation = SparkPlugin.extend({
           objectType: 'activity',
           clientTempId: uuid.v4(),
           actor: this.spark.internal.device.userId
-        });
+        }, pick(activity, 'activityType', 'parentActivityId'));
```

Comment:
You may want to modify some tests to assert these new properties are there, mostly to prevent them from being removed in future.

Reply:
@mattnorris I added an integration test there and it passed locally. Please check.